V2/api/mintPolygonNftFromAnyChain/src/index.ts

```diff
@@ -1,172 +1,178 @@
 // Import necessary libraries
-import { ethers } from "ethers";
-import axios from "axios";
+import { ethers } from 'ethers';
+import axios from 'axios';
+import { Hook, ChainType } from '@0xsquid/squid-types';

```

Comment:
I think you can import from the SDK directly, instead of needing the squid-types package
`import { Hook, ChainType } from "@0xsquid/sdk/dist/types";`